compress/jbig2_decompress_image.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
add comment on top

Reply:
This example also has nothing to do with PDF directly, so might make sense in
image/jbig2 subfolder

Reply:
or just jbig2/ subfolder

Reply:
I was thinking about it during creation time too. I think just jbig2/ subfolder would be good.